app/src/test/java/com/sdp13epfl2021/projmag/database/FirebaseProjectsDatabaseTest.kt

```diff
@@ -207,36 +209,15 @@ class FirebaseProjectsDatabaseTest {
         Mockito.`when`(mockQDS["tags"]).thenReturn(project.tags)
         Mockito.`when`(mockQDS["isTaken"]).thenReturn(project.isTaken)
         Mockito.`when`(mockQDS["description"]).thenReturn(project.description)
+        Mockito.`when`(mockQDS["videoUri"]).thenReturn(project.videoUri)
```

Comment:
This code is commented so there is no problem, but there is a typo : videoURI instead of videoUri.